Changelogs for Digital Twin Manager Plugins 4.2.0 (#3982)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1023-2-0-data-points-now-require-only-dtm-specific-permissions-ctm-2568-2464.md

---
date: ""

> title: "Data Points now require only DTM specific permissions [CTM-2568] (#2464)"

Reviewer: ```suggestion
title: "Data points now only require DTM permissions"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2568"
version: "1023.2.0"
---

> Previously, Data Points required both inventory and DTM permissions, but

Reviewer: ```suggestion
Previously, data points required both inventory and Digital Twin Manager (DTM) permissions. With this change, 
```

> this update removes the inventory-related permissions. Now, Data Points

Reviewer: ```suggestion
the inventory-related permissions have been removed. Data points
```

> now rely solely on the DTM permissions needed for its functionality.

Reviewer: ```suggestion
now only require the DTM permissions.
```
